### PR TITLE
Feature/volunteer page ux2

### LINKED ISSUE #665: [v2] Apply UX/UI on Join Us page
Skeleton and content are up but it's called "volunteer".

Please apply UX/UI design to this :)

### PR DESCRIPTION
**Pull request by ngottlieb**
waiting for the icon set from the design to finish this. Resolves #665

### CODE REVIEW

**Comment by ngottlieb** [comment]
Icons added. The icon set in the Figma design didn't match the actual text requiring icons from the content document...I sniped the Facebook logo for "social media" and then used the pointy finger from the design for outreach.

ready for review @linderr

File: views/volunteer.handlebars [hunk 1]
```diff
@@ -1,113 +1,95 @@
...
+        add more usability around our data, and we welcome your thoughts and comments on how we can improve!
+
+      </p>
+    </section>
+    <section>
+      <h4>
+        <span class="page-subhead">Can you help?</span>
+        Our Immediate Volunteer Needs:
+      </h4>
+      <!-- NOTE: just add "d-none" to the class of these divs to hide them !-->
```
Comment (linderr) [inline views/volunteer.handlebars:24]: Are we able to add a bit more space here?

**Comment by ngottlieb** [comment]
@linderr I can't find your comment asking for more space anymore but I added it

File: public/images/icons/addlroles.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...764C20.9739 23.711 20.7516 23.1679 20.3536 22.7665C19.9555 22.365 19.4144 22.1381 18.849 22.1355H20.9004C21.1802 22.1355 21.4572 22.0804 21.7156 21.9734C21.9741 21.8663 22.2089 21.7094 22.4068 21.5116C22.6046 21.3138 22.7615 21.0789 22.8685 20.8205C22.9756 20.562 23.0307 20.285 23.0307 20.0053V19.9503C23.0319 19.6612 22.976 19.3746 22.8662 19.1071C22.7564 18.8396 22.5949 18.5964 22.3909 18.3914C22.187 18.1864 21.9446 18.0237 21.6776 17.9125C21.4107 17.8014 21.1244 17.744 20.8353 17.7437L22.8022 17.7429C23.3591 17.7519 23.8997 17.5547 24.3201 17.1894C24.7405 16.824 25.011 16.3161 25.0796 15.7633C25.1097 15.4584 25.0757 15.1505 24.9797 14.8596C24.8837 14.5686 24.7278 14.3009 24.5221 14.0738C24.3164 13.8467 24.0654 13.6652 23.7853 13.5409C23.5052 13.4166 23.2022 13.3524 22.8958 13.3522L32.3923 13.352C32.7348 13.352 33.0739 13.2845 33.3903 13.1534C33.7066 13.0224 33.9941 12.8303 34.2363 12.5881C34.4784 12.346 34.6705 12.0585 34.8015 11.7421C34.9326 11.4258 35 11.0867 35 10.7442C35 10.4018 34.9325 10.0627 34.8015 9.74628C34.6704 9.4299 34.4783 9.14243 34.2362 8.90029C33.994 8.65815 33.7066 8.46607 33.3902 8.33502C33.0738 8.20398 32.7347 8.13654 32.3922 8.13654V8.13655Z" fill="#3E54A1"/>
```
Comment (nburt) [inline public/images/icons/addlroles.svg:2]: we've been data urls embedded in the css to minimize the number of requests we make. Can you move to icons.data.svg.css?

File: sass/style.css [hunk 3]
```diff
@@ -379,8 +379,53 @@ h3.city {
   opacity: 0.4;
 }
 
-p.page-info {
-  width: 60%;
+/*********************
```
Comment (nburt) [inline sass/style.css:382]: OK with waiting for now but now that we have webpack we should switch to sass so we can use variables

Comment (ngottlieb) [reply sass/style.css:382]: I've actually been writing scss in this file and webpack is compiling it, hahah. We should definitely change it to `.scss`, I hadn't taken the time to check if webpack would compile that without config changes

File: public/images/icons/addlroles.svg (hunk 2 above)
Comment (ngottlieb) [reply public/images/icons/addlroles.svg:2]: k, I'll give that a go

**Comment by ngottlieb** [comment]
Note (to self and others): still need to move the icon SVGs into data URLs in the CSS, but have to head out right now. will finish later tonight

**Comment by linderr** [comment]
Love using the do not merge label to keep a book mark on things!

**Comment by saikatbhadra** [comment]
What's supposed to be in the right column in this new view? I don't see anything

**Comment by saikatbhadra** [comment]
Looks good but I am unable to see 2 panels on my macbook air (relatively low resolution) when I go to the join us sub page. I see only content on the left panel

**Comment by ngottlieb** [comment]
@saikatbhadra the content and the design weren't really matched up -- only two short paragraphs of text that would've gone in the left panel, so after some discussion in Slack I decided to make it one column instead. I would've gone full-width, but the designs for the other one-column static page (About) had it only using a portion of the width, so I mimicked that

**Pull request merged**